espanol/canudos.md

```diff
@@ -105,3 +105,7 @@ Conectó su guitarra al amplificador y se puso a tocar una versión de "Siempre
 A pero si no leiste el cuento del cuco, entonces significa que no tienes [miedo] (miedo/miedo.md)
 
 [Puso su mano derecha en su frente y se teletransportó al centro del ejército enemigo](pelea_epica/pelea_epica.md)
+
+Lo que nadie sabía, es que el consejero era un versado maestro instruído en las nobles artes Jedi. 
+[Dark_side](dark_side/dark_side.md)
+[Light_side](light_side/light_side.md)
```

Comment:
There is a link on this line and also on the previous line.  Please make sure to insert a blank line between links.  That will help keep options visually separated.